resources/assets/uad_lists.json

```diff
@@ -15920,6 +15920,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.boundax.koreapreloadappinstaller",
+    "list": "Oem",
+    "description": "서비스 앱 설치 (Install the Services app), Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```

Comment:
Could you also translate the Korean part (and use another wording that's between the parentheses)?

Reply:
is translated on the parenthesis the Korean works, do you mean that it should be in quotes?

Reply:
> is translated on the parenthesis the Korean works, do you mean that it should be in quotes?

Remove the Korean part and only keep the English string. I don't get how "Install the Services app" is the name of the app. The first sentence should always be the name of the app.